src/App/App.js

```diff
@@ -1,6 +1,8 @@
 const App = () => (
   <div className="App">
-    <h1>Hello, Paulinho</h1>
+    <h1>Hello, Outra coisa</h1>
+
+    <h2>Teste2 </h2>
```

Comment:
Para quê esse espaço? Poderia remove-lo e ter o mesmo resultado.
```suggestion
    <h2>Teste2</h2>
```

Reply:
Realmente precisa disso?

Reply:
Precisa sim